app/config/collections.php

```diff
@@ -1,12 +1,15 @@
 <?php
 
-use Appwrite\Auth\Auth;
 use Utopia\Config\Config;
-use Utopia\Database\Database;
-use Utopia\Database\Helpers\ID;
 
-$providers = Config::getParam('oAuthProviders', []);
+$common = include __DIR__ . '/collections/common.php';
+$buckets = include __DIR__ . '/collections/buckets.php';
+$projects = include __DIR__ . '/collections/projects.php';
+$databases = include __DIR__ . '/collections/databases.php';
+$platform = include __DIR__ . '/collections/platform.php';
+
 $auth = Config::getParam('auth', []);
+$providers = Config::getParam('oAuthProviders', []);
```

Comment:
I dont see those being used. Do we need them? Did we have them before - do we need to move to some of the new files?

Reply:
We had it before, didn't find its usage but kept it as is, just in case...

Reply:
Let's try to remove and see if tests pass